partcad/src/partcad/document.py

```
@@ -0,0 +1,461 @@
+            '<th style="text-align: %s">%s</th>' % (align, html.escape(str(column)))
+            for column, align in zip(block.columns, aligns)
+        )
+        body = []
+        for row in block.rows:
+            # A row with more cells than there are columns keeps them all, the
+            # same way markdown does: the three formats have to say the same
+            # thing about the same document.
+            row_aligns = list(aligns) + ["left"] * max(0, len(row) - len(aligns))
+            cells = "".join(
+                '<td style="text-align: %s">%s</td>' % (align, html.escape("" if cell is None else str(cell)))
+                for cell, align in zip(row, row_aligns)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)